src/mutableAtom/index.ts

```diff
@@ -0,0 +1,138 @@
+import { atom } from 'jotai'
+import type { PrimitiveAtom } from 'jotai'
+import { proxy, snapshot, subscribe } from 'valtio'
+import type {
+  Action,
+  ActionWithPayload,
+  Options,
+  ProxyState,
+  Store,
+  Wrapped,
+} from './types'
+
+export function mutableAtom<Value>(
+  value: Value,
+  options: Options<Value> = defaultOptions
+) {
+  const valueAtom = atom({ value })
+
+  if (process.env.NODE_ENV !== 'production') {
+    valueAtom.debugPrivate = true
+  }
+  return makeMutableAtom(valueAtom, options)
+}
+
+export function makeMutableAtom<Value>(
+  valueAtom: PrimitiveAtom<Wrapped<Value>>,
+  options: Options<Value> = defaultOptions
+) {
+  const { proxyFn } = { ...defaultOptions, ...options }
+
+  const storeAtom = atom<
+    Store<Value>,
+    [ActionWithPayload<'setValueAsync', Value> | Action<'mount'>],
+    void
+  >(
+    (get, { setSelf }) => ({
+      proxyState: null,
```

Comment:
I think we can create a new proxy state here.

Reply:
I'm struggling to get proxy({ value: get(valueAtom) }) into the store.
This causes the store to recalculate on value change, but then it would need to subscribe / unsubscribe every time value changes.

Reply:
Okay, so, it's not here. 
I don't prefer `getValue: () => get(valueAtom).value,` in that sense. 🤔

Reply:
Hmm, I'm not sure what you mean.

When the mutable atom is first read it creates a proxy and saves it to the store. On next read, the existing proxy is used.

Proxy is needed on first read. This can happen before the first atom mount event. So the only way I can think to synchronously create the proxy once on first read is to check the static store for its existence.

Reply:
See: https://github.com/dmaskasky/jotai-valtio/pull/1